rem/aufile/aufile.c

```diff
@@ -307,8 +307,12 @@ int aufile_set_position(struct aufile *af, const struct aufile_prm *prm,
 	off_t pos = (off_t)(prm->srate * aufmt_sample_size(prm->fmt)
 		* prm->channels * pos_ms / 1000);
 
+	pos = min((off_t)datasize, pos);
```

Comment:
`SEEK_CUR` sets the position relative to the current position.

What is `datasize`?

Reply:
The size of the data chunk, i.e. the size of the PCM data in bytes.

Reply:
The `datasize` was 1082 instead of 1072 - the file was 67.625 ms long. I've cut it to 67ms exactly now

Reply:
I don't understand why the windows tests are failing, I'll try to reproduce

Reply:
> The `datasize` was 1082 instead of 1072 - the file was 67.625 ms long. I've cut it to 67ms exactly now

So, now you fixed the WAV file? Shouldn't we test that it doesn't read too far?

Reply:
Yes, that seemed simpler than fixing the test (i.e. allowing that less than 1ms can be read).

I can extend the test to make sure we don't read too far, that would make sense too.